src/sentry/api/endpoints/relay_projectconfigs.py

```diff
@@ -80,8 +85,12 @@ def _post(self, request):
             with Hub.current.start_span(op="get_config"):
                 with metrics.timer("relay_project_configs.get_config.duration"):
                     project_config = config.get_project_config(
-                        project, org_options=org_opts, full_config=full_config_requested
+                        project,
+                        org_options=org_opts,
+                        full_config=full_config_requested,
+                        project_keys=project_keys.get(project.id, []),
                     )
-                configs[six.text_type(project_id)] = project_config.to_camel_case_dict()
+
+            configs[six.text_type(project_id)] = project_config.to_camel_case_dict()
```

Comment:
Would be interesting to see a timing for that .

Reply:
I undid this because virtually no time was spent in it.